internal/services/workers_script/resource.go

```diff
@@ -304,9 +304,8 @@ func (r *WorkersScriptResource) Read(ctx context.Context, req resource.ReadReque
 		content = string(bytes)
 	}
 
-	// only update `content` if it was already present in state
-	// which might not be the case if `content_file` is used instead
-	if !data.Content.IsNull() {
+	// only update `content` if `content_file` isn't being used instead
+	if data.ContentFile.IsNull() {
```

Comment:
Previously, the implementation assumed that `content` would only be null in state if `content_file` is being used instead. However, when importing a resource both `content` and `content_file` are null in state. Instead, we should just populate `content` _unless_ `content_file` is present.